webhooks-extension/test/cleanup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export tekton_repo_dir=$(git rev-parse --show-toplevel)
+export test_dir="${tekton_repo_dir}/webhooks-extension/test"
+
+source ${test_dir}/config.sh
+source ${test_dir}/util.sh
+
+if [ $DASHBOARD_INSTALL_NS != "default" ]; then
```

Comment:
Is it ok to potentially delete the namespace with the deployment inside of it?

Reply:
Deleting the namespace deletes everything in that namespace as well.